libs/ui-toolkit/src/components/trading-select/select.tsx

```diff
@@ -175,3 +177,97 @@ export const MiniSelectOption = forwardRef<
     <SelectPrimitive.ItemText>{children}</SelectPrimitive.ItemText>
   </SelectPrimitive.Item>
 ));
+
+type MultiSelectProps = React.ComponentProps<typeof DropdownPrimitive.Root> & {
```

Comment:
This new component uses a different radix primitive so I think it would be better in its own directory or with the other TradingDropdowns